src/main/kotlin/no/nav/bidrag/inntekt/service/AinntektService.kt

```diff
@@ -96,7 +96,8 @@ class AinntektService() {
     // Grupperer og summerer poster som har samme kode/beskrivelse
     private fun grupperOgSummerDetaljposter(
         inntektPostListe: List<InntektPost>,
-        multiplikator: Int = 1, // Avviker fra default hvis beløp skal regnes om til årsverdi
+        // Avviker fra default hvis beløp skal regnes om til årsverdi
+        multiplikator: Int = 1,
     ): List<InntektPost> {
```

Comment:
Synes egentlig det er mer oversiktilig å ha kommentaren på samme linja om det er plass. Mener jeg løste dette ved å legge inn _ktlint_standard_discouraged-comment-location = disabled_ i .editorconfig, men mulig det ikke fungerer...?

Reply:
![image](https://github.com/navikt/bidrag-inntekt/assets/104764765/c92ec93c-aaff-4725-af1e-0477c21fccd0)

Får opp feilmelding om at dette ikke er støttet i .editorconfig. Ktlint klagde på denne ved kjøring av antrun:run@ktlint-format

Reply:
Ser ut som dette virket i versjon 1.0.1, men ikke lenger fra 1.1.0, så da kan den vel likegodt fjernes: https://github.com/pinterest/ktlint/issues/2445. Men ser også det står 'The property is not supported' på to av de andre reglene som ligger i .editorconfig

Reply:
comment propertien hadde ingen funksjon, men de to andre hadde det selv om det står de ikke gjør det. Lar de ligge.